nerfstudio/scripts/datasets/process_project_aria.py

```diff
@@ -219,6 +221,21 @@ def main(self) -> None:
             "fisheye_crop_radius": rgb_valid_radius,
         }
 
+        # save global point cloud, which is useful for Gaussian Splatting.
+        points_path = self.mps_data_dir / "global_points.csv.gz"
+        if points_path.exists():
```

Comment:
You should always filter the points from the aria point cloud --> here is a simple way to do it: https://facebookresearch.github.io/projectaria_tools/docs/data_utilities/core_code_snippets/mps#point-cloud

Reply:
We also have LOAD interface for any MPS data, no need to include pandas

i.e
```
points_data = mps.read_global_point_cloud(points_path) if points_path else None
# Filter out low confidence points
points_data = filter_points_from_confidence(points_data)
# Down sample points (optional)
# points_data = filter_points_from_count(points_data, 500_000)
# Retrieve point position
point_positions = [it.position_world for it in points_data]
```

Reply:
done!!

filtered points (red) look reasonable:
![image](https://github.com/nerfstudio-project/nerfstudio/assets/6992947/7d6b5ff8-2208-4c5f-ba29-1a9d637283f1)